AdminWebsite/AdminWebsite/ClientApp/src/app/work-allocation/edit-work-hours/vho-work-hours-table/vho-work-hours-table.component.spec.ts

```diff
@@ -382,5 +375,14 @@ describe('VhoWorkHoursTableComponent', () => {
             expect(result).toBe(false);
             expect(videoServiceSpy.setVhoNonAvailabiltiesHaveChanged).toHaveBeenCalledTimes(1);
         });
+
+        it('should display a message', () => {
+            // arrange
+            const Message = 'Tesing';
+            // act
+            component.message = Message;
+            // assert
+            expect(component.message).toBe('Tesing');
+        });
     });
```

Comment:
This isn't testing anything

Reply:
Hi Will its testing the behaviour of the variable was introduced . 
line 381 assigning a variable to Message  line 383 assigning hard coded variable to component.Message 
and expecting that to equal to message .